### ISSUE TITLE
depth

### ISSUE THREAD

**Issue by caiyongqi** [liuyuan-pal/NeuRay#20]
Hi. In the DTU dataset, I found that there is a big difference between the depth map estimated using colomap (you provided) and the ground truth depth map, have you checked this?

scan1, view 0, 300x400:
ground truth depth map:
![image](https://user-images.githubusercontent.com/47025383/199051600-f9849304-daaa-4649-bde9-2d72260e4dae.png)
depth map estimated using colomap:
![image](https://user-images.githubusercontent.com/47025383/199051735-eacb4ba9-30bc-4de1-b447-665cfc2afc52.png)

Could you check it? Thanks!!!

**Comment by liuyuan-pal** [comment]
Does this refer to the training set of NeuRay? As for the DTU training dataset, I adopt the processed dataset from PixelNeRF and run the colmap based on this processed version. I'm not sure what cause the difference. It may be caused by the normalization of scale and offset. Since COLMAP indeed will produce some incorrect depth in test time, it would be better to let the model receive these incorrect depth values in training.

**Comment by caiyongqi** [comment]
> Does this refer to the training set of NeuRay? As for the DTU training dataset, I adopt the processed dataset from PixelNeRF and run the colmap based on this processed version. I'm not sure what cause the difference. It may be caused by the normalization of scale and offset. Since COLMAP indeed will produce some incorrect depth in test time, it would be better to let the model receive these incorrect depth values in training.

OK, Thanks!  Could you update how to use colomap to get depth map (code or document) if you have time? I believe it will be useful.

**Comment by liuyuan-pal** [comment]
You may refer to the document of COLMAP. It already provides `patch_match_stereo` to estimate depth maps and you can read these depth maps from the provided python function like https://github.com/liuyuan-pal/NeuRay/blob/266af33b189ff6cfd7c6ae87a296a208b0219446/dataset/database.py#L129
